### PR TITLE
#1 Updates to grammar and spelling

### PR DESCRIPTION
**Pull request by BlueCodeThree**
For issue #1 

I tried to update some of it to make it clearer, but I don't know if I achieved that. 

I added a navigable table of contents. You also had some elements in your table of contents that wasn't actually in the body of the readme, so I added these headers in. 

Also, may I recommend the "code spell checker" plugin for vs code - great for catching typos and misspellings.

### CODE REVIEW

**Comment by Roakz** [comment]
Why would I install the spell checker when I can just create issues and let other people deal with it...giving them a sense of worth through contribution :P

**Comment by Roakz** [comment]
a couple of small changes but I appreciate the improvments!

**Comment by BlueCodeThree** [comment]
> Why would I install the spell checker when I can just create issues and let other people deal with it...giving them a sense of worth through contribution :P

I only did this because I was frustrated with something else I was working on :stuck_out_tongue_closed_eyes:

**Comment by BlueCodeThree** [comment]
Also, as you go through the code, you can leave comments directly on what you would like changed. Just in case you don't know. Makes replying to issues easier, will be handy when it gets to the actual code.

File: README.md [hunk 1]
```diff
@@ -1,79 +1,90 @@
 # myCitySubscription
 
-![Logo](./assets/mycity.png)
+![myCitySubscription Logo](./assets/mycity.png)
 
-A subscription based donation software for non for profits and the likes.
```
Comment (BlueCodeThree) [inline README.md]: Not sure how "and the likes" adds personality.

Comment (BlueCodeThree) [reply README.md]: But now I can comment on this, and also I see another error I missed!

**Comment by BlueCodeThree** [comment]
how's that?

**Comment by Roakz** [comment]
I'm not sure on your semicolon usage it doesn't sit right grammatically with the way the expressions roll of the tongue lol but I can't be bothered going backward and forwards over a semicolon so let's merge this thing and ill change it later

**Pull request merged**